digiwf-cypress/cypress/e2e/TL-5-Vorgaege-Anzeigen.cy.js

```diff
@@ -0,0 +1,104 @@
+import vorgangStarten from "../pages/vorgangStarten"
+import meineAufgaben from "../pages/meineAufgaben"
+import exampleUserTask from "../pages/exampleUserTask"
+import userTask from "../pages/userTask"
+import {EXAMPLE_USER_TASK_KEY} from "../constants/dataElementKeys"
+import {EXAMPLE_USER_TASK_NAME} from "../constants/dataElementKeys"
+import {FULL_USER_NAME} from "../constants/environmentVariables"
+import aktuelleVorgaenge from "../pages/aktuelleVorgaenge"
+import workflowUserTask from "../pages/workflowUserTask"
+
+const numberOfTasks = 21
+
+before(() => {
+    cy.login();
+})
+
+
+describe('Vorgaenge Anzeigen', () => {
+    it('passes', () => {
+        let pageSize = 20;
+
+        // Step 0: Create User processes
+        cy.log('Step 0');
+        meineAufgaben.openVorgangStarten();
+        vorgangStarten.findProcess(EXAMPLE_USER_TASK_NAME);
+        createTasks(numberOfTasks-1);
+        //ensures all the tasks are loaded
+        vorgangStarten.openMeineAufgaben();
+        reloadPageUntilTasksVisible(numberOfTasks-1);
+        closeTasks(numberOfTasks-1);
+
+        //create tracked task
+        meineAufgaben.openVorgangStarten();
+        vorgangStarten.getSearchField().should('be.visible');
+        vorgangStarten.findProcess(EXAMPLE_USER_TASK_NAME);
+        createTasks(1)
+
+        //Step 1: Open Aktuelle Vorgaenge
+        cy.log('Step 1');
+        vorgangStarten.openAktuelleVorgaenge();
+
+        //Step 2-5: Test different page sizes
+        cy.log('Step 2-5');
+
+        pageSize = 5;
+        aktuelleVorgaenge.changePageSize(pageSize);
+        aktuelleVorgaenge.checkPageSize(pageSize,numberOfTasks);
+
+        pageSize = 10;
+        aktuelleVorgaenge.changePageSize(pageSize);
+        aktuelleVorgaenge.checkPageSize(pageSize,numberOfTasks);
+
+        pageSize = 20;
+        aktuelleVorgaenge.changePageSize(pageSize);
+        aktuelleVorgaenge.checkPageSize(pageSize,numberOfTasks);
+
+        //Step 6-7: Check one list element
+        cy.log('Step 6-7');
+        aktuelleVorgaenge.findProcess(EXAMPLE_USER_TASK_NAME);
+        aktuelleVorgaenge.getElement(1).click();
+        workflowUserTask.taskIsCorrect(1,"User Task");
+        workflowUserTask.getStatus(1).should('contain.text','offen');
+        workflowUserTask.openMeineAufgaben();
+        closeTasks(1)
+        meineAufgaben.openAktuelleVorgaenge()
```

Comment:
Schlägt interessanterweise fehl, wenn man ganz am Anfang noch einen anderen Task hat. Bin mir jetzt nicht mehr ganz sicher was du gesagt hast ob du immer von 0 aktuellen Tasks ausgehst oder nicht?

Reply:
Ja das liegt an der Funktion reloadPageUntilTasksVisible. Ich kann das auch abändern, wenn das gewünscht ist. Fand es nur so suberer.

Reply:
Ich glaube das muss man sich in Zukunft eh nochmal Gedanken machen auf was man Fokus legt und dann die Tests etwas umbauen aber ist ja super simpel da deine Arbeit einfach minimal anzupassen.

Passt für mich dann so.